internal/kafka/test/integration/kafkas_test.go

```diff
@@ -662,6 +673,23 @@ func TestKafkaCreate_TooManyKafkas(t *testing.T) {
 
 	Expect(err).To(HaveOccurred(), "Expecting error to be thrown when creating more kafkas than allowed by the cluster limit:  %v", err)
 	Expect(resp.StatusCode).To(Equal(http.StatusForbidden))
+
+	clusterCriteria := services.FindClusterCriteria{
+		Provider: "aws",
+		Region:   "us-east-1",
+		MultiAZ:  true,
+	}
+
+	defer func() {
+		kasfFleetshardSync.Stop()
+		h.Env.Stop()
+
+		list, _ := test.TestServices.ClusterService.FindAllClusters(clusterCriteria)
+		for _, clusters := range list {
+			getAndDeleteServiceAccounts(clusters.ClientID, h.Env)
+			getAndDeleteServiceAccounts(clusters.ID, h.Env)
```

Comment:
These functions does not return anything but, in case of error, are invoking `panic`. Invoking `panic` into tests is usually discouraged.

I thing it would be better to change them to return the error, and change this to:
```suggestion
			if err := getAndDeleteServiceAccounts(clusters.ClientID, h.Env); err != nil {
			    t.Fatal(......)
			}
			if err := getAndDeleteServiceAccounts(clusters.ID, h.Env); err != nil {
			    t.Fatal(......)
			}
```